code/datums/status_effects/xeno_buffs.dm

```diff
@@ -211,9 +203,8 @@
 
 /// Updates the link's appearance.
 /datum/status_effect/stacking/essence_link/proc/update_beam()
-	var/beam_alpha = round(255 / (max_stacks+1 - stacks)) // 255 is the maximum value possible. We divide that by the missing amount of stacks.
-	if(current_beam)
-		current_beam.visuals.alpha = beam_alpha
+	var/beam_alpha = round(255 / (max_stacks+1 - stacks))
```

Comment:
Uneeded var

Reply:
Copypasting this from the earlier reviews:
There's no real need to put the math in the proc, though.

Reply:
Done in next commit.